testing/kfctl/kfctl_go_test.py

```diff
@@ -71,10 +80,9 @@ def test_build_kfctl_go(app_path, project):
   # username and password are passed as env vars and won't appear in the logs
   run_with_retries([kfctl_path, "init", app_path, "-V", "--platform=gcp",
                          "--version=" + version,
```

Comment:
should the PULL_NUMBER be used to fetch that PR from github
eg `kfctl init ... --version pull/os.getenv("PULL_NUMBER")

Reply:
Isn't that what's happening? 
Although once https://github.com/kubeflow/kubeflow/issues/2831 we should just pull the configs from the local checkout of the source.